.gitlab-ci.yml

```
@@ -104,7 +101,7 @@ test:debug:
   script:
     - rm -rf ~/.gradle/daemon/
     - export DD_AGENT_HOST="$BUILDENV_HOST_IP"
-    - GRADLE_OPTS="-Xmx3072m" ./gradlew :unitTestDebug --stacktrace --no-daemon --build-cache --gradle-user-home cache/
+    - GRADLE_OPTS="-Xmx3072m" ./gradlew :unitTestDebug --stacktrace --no-daemon --build-cache --gradle-user-home cache/ -Dorg.gradle.jvmargs=-javaagent:$DD_TRACER_FOLDER/dd-java-agent.jar=dd.env=ci,dd.trace.enabled=false,dd.jmx.fetch.enabled=false,dd.tags=test.configuration.variant:debug
```

maybe we can extract the common part of agent arguments inside a variable to be stored in the `variables` section do avoid the repetition. Ex. `DD_COMMON_AGENT_CONFIG="-Dorg.gradle.jvmargs=-javaagent:$DD_TRACER_FOLDER/dd-java-agent.jar=dd.env=ci,dd.trace.enabled=false,dd.jmx.fetch.enabled=false"`, so that the line looks like:

```suggestion
    - GRADLE_OPTS="-Xmx3072m" ./gradlew :unitTestDebug --stacktrace --no-daemon --build-cache --gradle-user-home cache/ -D$DD_COMMON_AGENT_CONFIG -Ddd.tags=test.configuration.variant:debug
```

---

Makes sense, done

---

That didn't seem to work, I think because of referencing an env var defined inside the Docker image from an env var defined inside the pipeline. So I had to change it slightly, to only have the actual agent args inside the "common config" variable.